Add Dockerfile & fix Python 3 incompatibilities

jimjshields opened the pull request:
This PR:
* adds a Dockerfile to build an image for running `awslimits` on a container
* changes all relative imports to be explicit (e.g., `from .support import load_tickets` instead of `from support import load_tickets`) to work with Python 3
* adds `gunicorn` as a dependency for running in production
* bumps `Flask-WTF` to handle import failure in Python 3

To run:
```bash
docker build -t awslimits:test .
docker run {AWS env vars} -it awslimits:test python awslimits/server.py
# or, run with gunicorn locally or in production
docker run {AWS env vars} -it -p 5000:5000 awslimits:test gunicorn awslimits.server:app --bind 0.0.0.0:5000
```

Next steps:
* Publish image to Dockerhub
* Add deploy & run commands for Docker in Makefile

jimjshields committed:
Add Docker support

* Add Dockerfile
* Run app on 0.0.0.0
* Bump Flask-WTF to 0.14.3 to work with Python 3

jimjshields committed:
Add gunicorn to requirements for production deployments

jimjshields committed:
Fix imports to be explicit for Python 3 & gunicorn

hltbra commented on Dockerfile:
```diff
@@ -0,0 +1,13 @@
+# copy everything else after pip install to take advantage of docker image layers (cache)
+COPY . /app
+
+WORKDIR /app
```
Adding `CMD` + `EXPOSE` would be nice for documentation and useful if anybody is running this locally.

Suggestion:
```dockerfile
EXPOSE 5000
CMD ["gunicorn", "awslimits.server:app", "--bind", "0.0.0.0:5000"]
```

jimjshields committed:
Move Dynamo helper to avoid circular import

jimjshields commented:
@hltbra I added that, any concerns with merging this and releasing a new version?

I tested this with Docker locally with Python 2.7.10 and Python 3.8.1 and tested main functionality (all tabs & updating a limit). I also fixed a circular import that broke in Python 2, but for some reason didn't break in Python 3.

hltbra reviewed (approved):
No concerns, @jimjshields! 🚢

jimjshields committed:
Fix print statements for Python 3

jimjshields committed:
Add Changelog

jimjshields committed:
Release 2.2.0
* Add Dockerfile to build and run AWSLimits using Docker
* Fix Python 3 compatibility issues
* Add Changelog
